spring-cloud-skipper-platform-cloudfoundry/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-cloud-skipper-platform-cloudfoundry</artifactId>
+    <packaging>jar</packaging>
+    <name>Spring Cloud Skipper :: CloudFoundry Platform</name>
+    <parent>
+        <artifactId>spring-cloud-skipper-parent</artifactId>
+        <groupId>org.springframework.cloud</groupId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <spring-cloud-deployer-cloudfoundry.version>1.3.0.M6</spring-cloud-deployer-cloudfoundry.version>
+        <reactor.version>3.0.7.RELEASE</reactor.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-skipper-server-core</artifactId>
```

Comment:
The platform POMs probably wouldn't need the skipper server-core as a dependency. Maybe we just need `spring-cloud-skipper` to make use of the Skipper domain classes.